From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: cpu/hotplug: reexport cpu_smt_control
Patch-mainline: never, kabi
References: kabi

Commit b284909abad4 (cpu/hotplug: Fix "SMT disabled by BIOS" detection
for KVM) removed an export of cpu_smt_control. This made the kABI
checker complaining.

Reintroduce the export.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/cpu.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/cpu.c
+++ b/kernel/cpu.c
@@ -354,6 +354,7 @@ void __weak arch_smt_update(void) { }
 #ifdef CONFIG_HOTPLUG_SMT
 
 enum cpuhp_smt_control cpu_smt_control __read_mostly = CPU_SMT_ENABLED;
+EXPORT_SYMBOL_GPL(cpu_smt_control);
 
 void __init cpu_smt_disable(bool force)
 {
